Source/main.swift

```diff
@@ -0,0 +1,109 @@
+//
```

Comment:
Let's use the new copyright header such as the one in `AppDelegate.swift` in this PR.